kaldb/src/main/java/com/slack/kaldb/server/KaldbConfig.java

```diff
@@ -31,7 +31,6 @@ public class KaldbConfig {
   public static final int DEFAULT_ZK_TIMEOUT_SECS = 15;
 
   public static final String CHUNK_DATA_PREFIX = "log";
-  public static final long DEFAULT_ROLLOVER_FUTURE_TIMEOUT_MS = 30000;
```

Comment:
Are there any side effects of changing the default timeout value to 15 secs from 30 secs?

Reply:
There is not - the only time we used this value was during shutdown, and this timeout was wrapped in the shutdown that forced a shorter timeout of 15 seconds. This should just prevent all the errors we see when nesting a 30s timeout inside a 15s timeout.